libs/algorithms/include/hpx/parallel/container_algorithms/transform.hpp

```diff
@@ -340,3 +341,112 @@ namespace hpx { namespace parallel { inline namespace v1 {
             std::forward<Proj2>(proj2));
     }
 }}}    // namespace hpx::parallel::v1
+
+namespace hpx { namespace ranges {
+
+    template <class I, class O>
```

Comment:
Please use `typename` for template parameters.

Reply:
Missed it.